queries/lrb/src/main/java/storm/lrb/bolt/FileTollDataStore.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * #!
+ * %
+ * Copyright (C) 2014 - 2015 Humboldt-Universität zu Berlin
+ * %
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #_
+ */
+/*
```

Comment:
Please remove the double copyright statement (the second one below). There are other files like this as well...
Do you add the copyright header manually (or by your IDE)? Maven uses a plugin to add it (the to one), so you should not add one manually.